components/NewsroomYoutube.js

```diff
@@ -0,0 +1,96 @@
+import { useEffect, useState, useRef } from 'react';
+import TextTruncate from 'react-text-truncate'
+import ArrowLeft from './icons/ArrowLeft';
+import ArrowRight from './icons/ArrowRight';
+import TextLink from './typography/TextLink';
+import tempData from '../config/videos.json';
+import Heading from './typography/Heading'
+import Paragraph from './typography/Paragraph'
+
+export default function NewsroomYoutube({ className = '' }) {
+  const [videoData, setvideoData] = useState([]);
+  const videoContainer = useRef();
+  const videoref = useRef();
+  const shiftLeft = (e) => {
```

Comment:
@magicmatatjahu please check the implementation of the onClick scroll feature in the Video and Livestreams Section.